src/Actions/types.js

```diff
@@ -33,3 +33,11 @@ export const EMPLOYEE_ASSETS_REQUEST = 'EMPLOYEE_ASSETS_REQUEST';
 export const EMPLOYEE_ASSETS_SUCCESS = 'EMPLOYEE_ASSETS_SUCCESS';
 export const EMPLOYEE_ASSETS_FAILURE = 'EMPLOYEE_ASSETS_FAILURE';
 export const REQUEST_SELECT_VALUES = 'REQUEST_SELECT_VALUES';
+/**
+ * === REQUEST NEW IN BOX ALERT ===
+ *    Request inbox alerts
+ * === REQUEST NEW IN BOX ALERT ===
+ */
+export const REQUEST_INBOX_ALERT = 'REQUEST_INBOX_ALERT';
+export const REQUEST_INBOX_ALERT_SUCCESS = 'REQUEST_INBOX_ALERT_SUCCESS';
+export const REQUEST_INBOX_ALERT_FAILURE = 'REQUEST_INBOX_ALERT_FAILURE';
```

Comment:
You might want to make action names consistent. It seems `Request` is different than others.